### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mdczaplicki**
Changes some `Starlite` references to `Litestar`.

[//]: # "By submitting this pull request, you agree to:"
[//]: # "- follow [Litestar's Code of Conduct](https://github.com/litestar-org/.github/blob/main/CODE_OF_CONDUCT.md)"
[//]: # "- follow [Litestar's contribution guidelines](https://github.com/litestar-org/.github/blob/main/CONTRIBUTING.md)"

### Pull Request Checklist

[//]: # "Please review the [Litestar contribution guidelines](https://github.com/litestar-org/.github/blob/main/CONTRIBUTING.md) for this repository."

- [ ] New code has 100% test coverage
- [ ] (If applicable) The prose documentation has been updated to reflect the changes introduced by this PR
- [ ] (If applicable) The reference documentation has been updated to reflect the changes introduced by this PR

### Description

[//]: # "Please describe your pull request for new release changelog purposes"

### Close Issue(s)

[//]: # "Please add in issue numbers this pull request will close, if applicable"
[//]: # "Examples: Fixes #4321 or Closes #1234"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Starlite sphinx theme
+# Litestar sphinx theme
 
-A Sphinx theme for the [Starlite organization](https://github.com/starlite-api), based on the
+A Sphinx theme for the [Litestar organization](https://github.com/starlite-api), based on the
```
Comment (peterschutt) [inline README.md]: Thanks for this, can you update the github URL here also please?

Comment (mdczaplicki) [reply README.md]: This one seems to be correct. It doesn't redirect to anything.

Comment (mdczaplicki) [reply README.md]: Nevermind. Got it.

**Comment by peterschutt** [comment]
Thanks!

**Pull request merged**